controllers/managedmcg_controller.go

```diff
@@ -457,20 +475,6 @@ func (r *ManagedMCGReconciler) removeNoobaa() error {
 		}
 	}
 
-	objectBucketClaims := &noobaav1alpha1.ObjectBucketClaimList{}
```

Comment:
Why this is removed? Need your help to understand the patch

Reply:
@v-harihar we added this code assuming that even if we remove the BucketClass OBC is not getting deleted in the uninstallation flow(we have code to delete OBC when respective BucketClass is deleted, bucketClassDeleted()). But the reason was mcg-osd CSV getting deleted without waiting for all the OBCs to delete. with the new PollImmediate() we are making sure all the OBCs and respective secret and Configmap are getting deleted.